Mathlib/RingTheory/Flat/Stability.lean

```diff
@@ -149,9 +150,9 @@ end BaseChange
 
 section Localization
 
-variable (R : Type u) (S : Type v) (M : Type w)
-  [CommRing R] [CommRing S] [Algebra R S]
-  [AddCommGroup M] [Module R M]
+variable (R : Type u) (S : Type v) (M Mp: Type w) (Rp : Type v)
```

Comment:
```suggestion
variable {R : Type u} {M Mp : Type*} (Rp : Type v)
```
They shouldn't share the same universes. `S` seems to be unnecessary, and they can be implicit.